src/test/java/com/fasterxml/jackson/databind/struct/TestUnwrappedWithDynamicPrefix.java

```diff
@@ -0,0 +1,205 @@
+package com.fasterxml.jackson.databind.struct;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonUnwrapped;
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.BeanProperty;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.JsonSerializer;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import com.fasterxml.jackson.databind.deser.ContextualDeserializer;
+import com.fasterxml.jackson.databind.deser.std.StdDeserializer;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.fasterxml.jackson.databind.ser.ContextualSerializer;
+import com.fasterxml.jackson.databind.ser.std.StdSerializer;
+import com.fasterxml.jackson.databind.testutil.DatabindTestUtil;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+public class TestUnwrappedWithDynamicPrefix extends DatabindTestUtil {
+    private final ObjectMapper MAPPER = newJsonMapper();
+    private final ObjectMapper CUSTOM_MAPPER = newJsonMapper()
+            .registerModule(new SimpleModule()
+                    .addSerializer(ValueAndMap.class, new ValueAndMapSerializer())
+                    .addDeserializer(ValueAndMap.class, new ValueAndMapDeserializer()));
+
+    @JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
Bad idea for tests -- should almost never be used: often masks legitimate problems (in production use useful for defensive handling).

Reply:
Yes, I agree. Removing this, however, will throw a `UnrecognizedPropertyException`

> Unrecognized field "prop1_l10n"

The idea was to see what the configuration was actually doing.

Reply:
Right, so it did hide a failure.